src/lib/tabmanager.js

```diff
@@ -9,17 +9,31 @@ TabManager.prototype = {
    * @param {ChromeWindow} chromeWindow
    * @returns {Object}
    */
-  getGroupsWithTabs: function(chromeWindow) {
+  getGroupsWithTabs: function(chromeWindow, sort) {
     let groups = this._storage.getGroups(chromeWindow);
     let tabs = this._storage.getTabs(chromeWindow);
 
-    return groups.map((group) => {
+    let retGroups = groups.map((group) => {
       return Object.assign({}, group, {
         tabs: tabs.filter((tab) => {
           return tab.group == group.id;
         })
       });
     });
+
+    if (sort) {
+      retGroups.sort((a, b) => {
+        if (a.title < b.title) {
+          return -1;
+        } else if (a.title > b.title) {
+          return 1;
+        } else {
+          return 0;
+        }
+      });
```

Comment:
`else` after a return statement is somewhat useless since it would never reach the code if the block above was true. You could simply omit the elses (which is was eslint suggests here). However, there is an even "nicer" (as in shorter) way of writing this sort code:

```js
retGroups.sort((a, b) => {
  if (a.title == b.title) {
    return 0;
  }

  return a.title < b.title ? -1 : 1;
});
```

I tend to avoid ternary operation, but in such simple cases as deciding between `1` and `-1`, they are perfectly valid and easy to understand.